metadata_backend/helpers/validator.py

```diff
@@ -2,15 +2,16 @@
 
 import re
 from io import StringIO
-from typing import Any, Dict, cast
+from typing import Any, Dict
 from urllib.error import URLError
 
 import ujson
 from aiohttp import web
+from defusedxml.ElementTree import ParseError
+from defusedxml.ElementTree import tostring as etree_tostring
```

Comment:
i don't think we needed to introduce here a new library i see xmlschema using: https://github.com/sissaschool/xmlschema/blob/master/xmlschema/__init__.py#L10